### PR TITLE
Create bellsofaba.json

### PR DESCRIPTION
**Pull request by bellsofaba**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8197"></a>

### CODE REVIEW

**Comment by bellsofaba** [comment]
Thanks for the opportunity!

File: data/bellsofaba.json [hunk 1]
```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Jack Aba",
+  "displayStatsPublic": true,
```
Comment (sandramsc) [inline data/bellsofaba.json:3]: The displayStatsPublic, avatar and type tags are no longer required, take a look at the [QuickStart Doc.](https://linkfree.io/docs/quickstart-json) if you haven't already.
screen

```suggestion
```

File: data/bellsofaba.json [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  "type": "personal",
```
Comment (sandramsc) [inline data/bellsofaba.json:4]: See above comment - tag no longer required.
```suggestion
```

File: data/bellsofaba.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  "bio": "**Software Developer**, Open Source Devotee, & Technical Writer",
+  "avatar": "https://github.com/bellsofaba.png",
```
Comment (sandramsc) [inline data/bellsofaba.json:6]: See above comment - tag no longer required.
```suggestion
```

**Review by sandramsc — CHANGES_REQUESTED** [review]

**Review by bellsofaba — COMMENTED** [review]
The recommended changes have been taken care of.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for Contributing! 🎉 

<p align="center"><a href="https://linkfree.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**